### ISSUE TITLE
Dial Color

### ISSUE THREAD

**Issue by reefland** [fineemb/lovelace-thermostat-card#11]
The dial background color is always orange because heat mode is enabled.   If I disable heat mode the dial background color is black.

I'd like the dial background to reflect the climate sensor's "hvac_action" attribute current value.

- "idle" shows black background.
- "heating" shows orange background.
- "cooling" shows blue background.

Essentially, exactly what happens when you look at a Nest thermostat.

**Comment by Quotacious** [comment]
I was wondering why all 3 of mine were always orange (I also have the Nest thermostats). :)

**Comment by fineemb** [comment]
Background color is currently using `preset_mode` this property
Icon color is currently using `hvac_state` this property
But my climate doesn't exist `preset_mode` or `hvac_action`  properties.
Can i provide the following screenshots?
![01](https://user-images.githubusercontent.com/16739914/74502802-a47b0300-4f29-11ea-9e98-989176b09c47.jpg)

**Comment by reefland** [comment]
Would be helpful if the configuration could let us override the default attribute being used.

Here are screen shots showing idle and heating in the "hvac_action" attribute. 
![ha_climate_attributes_heating](https://user-images.githubusercontent.com/16230914/74535789-95ce2500-4f04-11ea-9420-0e3b096293b2.png)

![ha_climate_attributes_idle](https://user-images.githubusercontent.com/16230914/74535790-95ce2500-4f04-11ea-8ab8-b2dc3116078f.png)

**Comment by fineemb** [comment]
hvac_action is it always the same as state? 
cooling(cool), heating(heat).....

**Comment by reefland** [comment]
No, state for me is always "heat" or "cool" or "off" or "auto" depending on the mode the thermostat is set for.  The "hvac_action" reflects what is happening this moment.  "idle", "heating", "cooling".

**Comment by fineemb** [comment]
Thanks, i already understand
I will find a way to solve this problem

**Comment by mycrouch** [comment]
Is there a way to also do a light theme?

![climate](https://user-images.githubusercontent.com/59326901/108217725-a81a1900-717f-11eb-982d-7b15044e4994.jpg)

**Comment by pqpxo** [comment]
Hi there!

I would also be interested in how to change the overall colour of the dial. I've tried dabbling with the .js file and found what needs changing but the changes I make never persist

**Comment by TimBeekman** [comment]
+1 on the light theme

**Comment by TimBeekman** [comment]
For everyone who wants light mode:

```
card_mod:
  style: |
    @media (prefers-color-scheme: light) {
      .dial {
        --thermostat-off-fill: #F4F4F4;
      }
      .dial text, .dial text tspan, .dial__lbl--target, .dial__lbl--low, .dial__lbl--high, .dial__lbl--ambient, .dial__lbl--super--ambient, .dial__lbl--super--target, .dial__lbl--super--high, .dial__lbl--super--low, .dial__lbl--ring, .dial__lbl--title {
        fill: black;
      }
      .dial__ticks path, .dial__ticks path.active.large {
        fill: #BDBDBD;
      }
    }
```
